documentation/modules/vmware-prerequisites.adoc

```diff
@@ -7,8 +7,62 @@
 
 The following prerequisites apply to VMware migrations:
 
+* You must be logged in as a user with at least the minimal set of xref:vmware-privileges_{context}[VMware privileges].
 * You must install link:https://www.vmware.com/support/ws5/doc/new_guest_tools_ws.html[VMware Tools] on all source virtual machines (VMs).
 * If you are running a warm migration, you must enable link:https://kb.vmware.com/s/article/1020128[changed block tracking (CBT)] on the VMs and on the VM disks.
 * You must create a VMware Virtual Disk Development Kit (VDDK) image.
 * You must obtain the SHA-1 fingerprint of the vCenter host.
 * If you are migrating more than 10 VMs from an ESXi host in the same migration plan, you must increase the NFC service memory of the host.
+
+[discrete]
+[id="vmware-privileges_{context}"]
+== VMware privileges
+
+The following minimal set of VMware privileges is required to migrate virtual machines to {virt} with the {project-first}.
+
+[cols="2", options="header"]
+.VMware privileges
+|===
+|Privilege |Description
+2+|`Virtual machine.Interaction` privileges:
+|`Virtual machine.Interaction.Power Off`   |Allows powering off a powered-on virtual machine. This operation powers down the guest operating system.
+|`Virtual machine.Interaction.Power On`  |Allows powering on a powered-off virtual machine and resuming a suspended virtual machine.
+2+a|`Virtual machine.Provisioning` privileges:
+[NOTE]
+====
+All `Virtual machine.Provisioning` privileges are required.
```

Comment:
@sbeskin-redhat All the Provisioning privileges are listed, so why to add this note? also, it does not look good in the preview

Reply:
@mguetta1
1. I asked Sasha to add the note to make it clear to the user that for this category, the user can just select all, rather than manually going through and selecting each one.
2. The output of all NOTE and IMPORTANT etc looks bad. Not just here. But this is an issue with the transformation to XML, it's not something that we can fix ourselves.

Reply:
I understand, thanks for the explanation @stoobie